mrf24j.cpp

```diff
@@ -148,6 +149,41 @@ void Mrf24j::send16(word dest16, char * data) {
     write_short(MRF_TXNCON, (1<<MRF_TXNACKREQ | 1<<MRF_TXNTRIG));
 }
 
+void Mrf24j::send16(word dest16, char * data, byte length) {
```

Comment:
THe right way of doing this is not to copy the whole function, but to make this new function be the "primary" function, and have the existing send16() function do the strlen() and then call this function.  Duplication is never ok for this sort of thing.